### ISSUE TITLE
Default Gravatar icon is being used even though 'Force Default Gravatar Icons' is disabled

### ISSUE THREAD

**Issue by heyitsritesh** [AlexanderOMara/flarum-gravatar#10]
`Force Default Gravatar Icons` is disabled, but it's still replacing the avatars for all users who have not uploaded a custom avatar yet. Default Flarum avatars have stopped showing up. Only Gravatars are showing for all users.

**Comment by AlexanderOMara** [comment]
That's actually how it's supposed to work. I think you might have misunderstood the option. It's just an option for [Gavatar's force default](https://en.gravatar.com/site/implement/images/#force-default) (`f=y`) to the gravatar URL.

Using this option you can force an image like:

[![](https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50?d=identicon&f=y)](https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50?d=identicon&f=y)

Instead of whatever image a user might customize:

[![](https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50?d=identicon)](https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50?d=identicon)

Basically when this extension is enabled, all avatars go though Gravatar (unless a local one is uploaded). If/when a user sets a custom avatar, that will be used (unless you use this option), else they will get a default Gravatar avatar.

Since this feature is working as intended, I'm going to close this issue. If you want a new feature where Flarum default avatars are shown if a custom Gravatar one isn't configured please open a new issue and maybe we can come up with some way to do that.
